src/ex4_js-objects-part2/task-02.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
+function blankProto() {
+	const proto = {};
+	const obj = Object.create(proto);
```

Comment:
proto is not empty
![image](https://user-images.githubusercontent.com/17769359/66723919-0286c500-ee28-11e9-9dc3-d07367db1d73.png)